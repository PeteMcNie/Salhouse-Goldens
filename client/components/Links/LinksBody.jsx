import React from 'react'

import Container from 'react-bootstrap/Container'
import Row from 'react-bootstrap/Row'
import Col from 'react-bootstrap/Col'
import Image from 'react-bootstrap/Image'

function LinksBody () {
  return (
    <>
      <Container>
        <Row className='row'>
          <Col lg={6} className='leftCol'>
            <h2 className='bodyTitle'>Links</h2>
            <p>Our favourite links:</p>
            <ul>
              <li><p><a href='https://www.mistymoorgoldens.com/'>Mistymoor Golden Retrievers</a> (home of NZ CH Santamaria The Patriot, Shadow&apos;s Sire)</p></li>
              <li><p><a href='https://sandlewoodgoldens.com/'>Sandlewood Golden Retrievers</a> (home of CH Sandlewood Gamble With Gold, Sherry&apos;s Sire)</p></li>
              <li><p><a href='https://www.dogsnz.org.nz/breeds/golden-retriever'>New Zealand official breed standard</a> (breed standard for golden retrievers in New Zealand)</p></li>
              <li><p><a href='https://en.wikipedia.org/wiki/Golden_Retriever'>Wikipedia Golden Retrievers</a></p></li>
              <li><p><a href='http://cgrcnz.org/'>Central Golden Retriever Club</a></p></li>
              <li><p className='linkslastline'><a href='https://keridale.com/'>Keridale Golden Retrievers</a> (another good breeder&apos;s website in New Zealand)</p></li>
            </ul>
          </Col>
          <Col lg={6}>
            <div>
              <Image src="images/link1.jpg" fluid className='linksimage'/>
            </div>
          </Col>
        </Row>
      </Container>
    </>
  )
}

export default LinksBody
