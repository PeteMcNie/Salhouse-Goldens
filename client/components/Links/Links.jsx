import React from 'react'

import Navbar from '../NavbarComp'
import LinksBody from './LinksBody'

function Links () {
  return (
    <>
      <Navbar />
      <LinksBody />
    </>
  )
}

export default Links
