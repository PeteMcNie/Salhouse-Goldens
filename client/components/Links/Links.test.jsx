import React from 'react'
import { render } from '@testing-library/react'

import Links from './Links'

test('Links component matches snapshot', () => {
  const { container } = render(<Links />)
  expect(container).toMatchSnapshot()
})
