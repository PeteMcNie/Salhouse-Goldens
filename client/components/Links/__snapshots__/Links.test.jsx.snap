// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Links component matches snapshot 1`] = `
<div>
  <nav
    class="navbar navbar-expand-lg navbar-dark bg-dark fixed-top"
  >
    <a
      class="navbar-brand"
      href="/"
    >
      <img
        alt="Goldens retrievers, phoenix and kula"
        class="shlogo d-inline-block align-top"
        src="images/logo.jpeg"
      />
    </a>
    <button
      aria-controls="responsive-navbar-nav"
      aria-label="Toggle navigation"
      class="navbar-toggler collapsed"
      type="button"
    >
      <span
        class="navbar-toggler-icon"
      />
    </button>
    <div
      class="navbar-collapse collapse"
      id="responsive-navbar-nav"
    >
      <div
        class="ml-auto navbar-nav"
      >
        <a
          class="nav-link"
          data-rb-event-key="/"
          href="/"
        >
          Home
        </a>
        <div
          class="dropdown nav-item"
        >
          <a
            aria-expanded="false"
            aria-haspopup="true"
            class="dropdown-toggle nav-link"
            href="#"
            id="collasible-nav-dropdown"
            role="button"
          >
            Our Goldens
          </a>
        </div>
        <div
          class="dropdown nav-item"
        >
          <a
            aria-expanded="false"
            aria-haspopup="true"
            class="dropdown-toggle nav-link"
            href="#"
            id="collasible-nav-dropdown"
            role="button"
          >
            Grooming Your Golden
          </a>
        </div>
        <a
          class="nav-link"
          data-rb-event-key="/mylifewithanimals"
          href="/mylifewithanimals"
        >
          My Life With Animals
        </a>
        <div
          class="dropdown nav-item"
        >
          <a
            aria-expanded="false"
            aria-haspopup="true"
            class="dropdown-toggle nav-link"
            href="#"
            id="collasible-nav-dropdown"
            role="button"
          >
            About Us
          </a>
        </div>
        <a
          class="nav-link"
          data-rb-event-key="/links"
          href="/links"
        >
          Links
        </a>
        <a
          class="nav-link"
          data-rb-event-key="/contact"
          href="/contact"
        >
          Contact
        </a>
      </div>
    </div>
  </nav>
  <div
    class="belowNavbar"
  >
    <h2>
      Links
    </h2>
    <p>
      Our favourite links:
    </p>
    <p>
      <a
        href="https://www.mistymoorgoldens.com/"
      >
        Mistymoor Golden Retrievers
      </a>
      (home of NZ CH Santamaria The Patriot, Shadow's Sire)
    </p>
    <p>
      <a
        href="https://sandlewoodgoldens.com/"
      >
        Sandlewood Golden Retrievers
      </a>
      (home of CH Sandlewood Gamble With Gold, Sherry's Sire)
    </p>
    <p>
      <a
        href="https://www.dogsnz.org.nz/breeds/golden-retriever"
      >
        New Zealand official breed standard
      </a>
      (breed standard for golden retrievers in New Zealand)
    </p>
    <p>
      <a
        href="https://en.wikipedia.org/wiki/Golden_Retriever"
      >
        Wikipedia Golden Retrievers
      </a>
    </p>
    <p>
      <a
        href="http://cgrcnz.org/"
      >
        Central Golden Retriever Club
      </a>
    </p>
    <p>
      <a
        href="https://keridale.com/"
      >
        Keridale Golden Retrievers
      </a>
      (another good breeder's website in New Zealand)
    </p>
  </div>
  <div>
    <img
      class="img-fluid"
      src="images/link1.jpg"
    />
  </div>
</div>
`;
